app/lib/actions/katello/repository/create_root.rb

```diff
@@ -3,6 +3,9 @@ module Katello
     module Repository
       class CreateRoot < Actions::EntryAction
         def plan(root)
+          unless ::Katello::RepositoryTypeManager.find(root.content_type)
+            fail _("Content type %{content_type} is not enabled." % { content_type: root.content_type })
+          end
```

Comment:
I think this validation will handle this right?  https://github.com/Katello/katello/blob/master/app/models/katello/root_repository.rb#L83-L87   (we could maybe change the wording there)

Reply:
I'll probably change the wording. I want to make sure there's a distinction between available defined types and what's enabled on the proxy.